test/DropdownItemSpec.js

```diff
@@ -98,13 +98,6 @@ describe('<Dropdown.Item>', () => {
       .should.not.have.property('onSelect');
   });
 
-  it('does not pass onSelect to children', () => {
-    mount(<Dropdown.Item onSelect={() => {}}>Item</Dropdown.Item>)
```

Comment:
shouldn't remove, adding a displayName to the components would make this work I think